docs/explanation/logical-axiomatization.md

```diff
@@ -0,0 +1,101 @@
+# Logical axiomatization of classes & use of reasoning
+
+This explainer requires understanding of ontology classifications, please see "an ontology as a classification" section of the [introduction to ontologies documentation](../explanation/intro-to-ontologies.md) if you are unfamiliar with these concepts. 
+
+## What are logical axioms
+
+Logical axioms are relational information about a class that are primarily aimed at machines (as opposed to annotations like textual definitions which are primarily aimed at humans). These logical axioms allow reasoners to assist in and verify classification, lessening the development burden and enabling expressive queries.
```

Comment:
'Information' is an uncountable noun.
Logical axioms are relational statements about classes that are primarily aimed at machines. This is opposed to annotations like textual definitions which are primarily aimed at humans.